contracts/gateway-v1.yaml

```diff
@@ -3579,4 +3698,75 @@ components:
         resp:
           $ref: '#/components/schemas/RequestReference'
       xml:
-        name: PatientIdentificationResponse
\ No newline at end of file
+        name: PatientIdentificationResponse
+    HipInitiatedLinkRequest:
+      type: object
+      properties:
+        requestId:
+          type: string
+          format: uuid
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        meta:
+          type: object
+          properties:
+            X-CM-ID:
```

Comment:
X-CM-ID? why not just "cmId"? I am assuming this 12345-12345-12345-6@ncg or so?

Reply:
Also does it need to be in the meta attribute? why not a root element? consistent with other apis?

Reply:
ok will move it into root